### ISSUE TITLE
link for "Included Software" included in "Set up job" is invalid

### ISSUE THREAD

**Issue by arielelkin** [actions/virtual-environments#1992]
**Description**  
A clear and concise description of what the bug is, and why you consider it to be a bug.

**Area for Triage**:  
Apple

**Question, Bug, or Feature?**:  
Bug

**Virtual environments affected**  
- [ ] Ubuntu 16.04
- [ ] Ubuntu 18.04
- [ ] Ubuntu 20.04
- [x] macOS 10.15
- [ ] macOS 11.0
- [ ] Windows Server 2016 R2
- [ ] Windows Server 2019

The log reads:

```
Current runner version: '2.273.6'
Operating System
Virtual Environment
  Environment: macos-10.15
  Version: 20201103.2
  Included Software: https://github.com/actions/virtual-environments/blob/macos-10.15/20201103.2/images/macos/macos-10.15-Readme.md
```

but the link returns a 404

**Comment by maxim-lobanov** [comment]
Hello @arielelkin , thank you for your report.
That link is invalid and release is not available because we have found critical bug in new image and decided to rollback it.
Currently, rollback is finished and all builds should go to the old image and link should be correct

**Comment by arielelkin** [comment]
@maxim-lobanov this is still an issue in my current builds, here's the relevant part of the log:

```
Virtual Environment
  Environment: macos-10.15
  Version: 20201107.1
  Included Software: https://github.com/actions/virtual-environments/blob/macos-10.15/20201107.1/images/macos/macos-10.15-Readme.md
```

**Comment by maxim-lobanov** [comment]
Wow, I see the issue. `macos-10.15` in url should be `macOS-10.15`.

Thank you for report, we definitely need to fix it.

**Comment by maxim-lobanov** [comment]
I have merged PR #2035 with fix. The next image should have correct url

**Comment by maxim-lobanov** [comment]
Changes were deployed, documentation url should work as expected now
